testsuite/features/step_definitions/api_common.rb

```diff
@@ -91,7 +90,8 @@
 end
 
 Then(/^I should get at least one role that matches "([^"]*)" suffix$/) do |suffix|
-  refute(@roles.find_all { |el| el =~ /#{suffix}/ }.empty?)
+  refute(@roles.find_all { |el| el =~ /#{suffix}/ }
+.empty?)
```

Comment:
Why putting this into a new line? Do you by any chance know what cop complains about the old style?

Reply:
I don't remember now, but personally I don't like it, let's revert this, and disable the Cop that warns with this code style.